Mutenix Macroboard

m42e opened the pull request:
Please accept mu Pull Request for my Mutenix Macroboard.

Its main Focus is to easily mute/unmute during Teams calls, but can be adapted to other useful stuff.

- [Hardware](https://github.com/mutenix-org/hardware-macroboard): CERN Open Hardware Licence Version 2 - Strongly Reciprocal
- [Firmware](https://github.com/mutenix-org/firmware-macroboard): MIT
- [Software Host](https://github.com/mutenix-org/software-host): MIT

Hardware used is either a RPI2040 or NRF52840.

cuvoodoo commented:
The software source files, license file, copyright info, and license info are present. All perfect there.
The hardware source files, license file, and gerber export are also present. What is missing is just adding the copyright info (author+year) and license name (like spdx id) in the schematic and board layout, and exporting the pdf schematic.
If you complete that, then we are good to go :).

m42e commented:
@cuvoodoo I have added the information (as text variables in the board. If there is another way of doing so, I would need a hint). And added the pdf export of the schematics.

Thanks a lot.

cuvoodoo commented:
if you look at the schematic page (or when editing the board layout), you can see a page border and a section in the bottom left corner (with the text "file" and so on). This is called a title block, and where you can set the title, copyright, and license info. it would then be directly visible (as opposed to variables), and the usual place hardware engineer would look for.
you can access it in kicad under file -> page settings. could you out the information there and re-export the schematic pdf?

m42e commented:
Done and thanks.
